nwc_backend/__init__.py

```diff
@@ -58,6 +58,18 @@ def ready() -> str:
 
     app.register_blueprint(frontend_api_bp, url_prefix=base_path + "api")
 
+    @app.route(f"{base_path}config.js")
+    async def serve_config() -> Response:
+        return Response(
+            f"""window.NWC_CONFIG = {{
+    UMA_VASP_LOGIN_URL: "{app.config['UMA_VASP_LOGIN_URL']}",
```

Comment:
Can you add a brief note explaining why this is needed? I see Future Joel or Jeremy looking at this and going WTF, why is this here?